### ISSUE TITLE
Can't open tabs groups

### ISSUE THREAD

**Issue by techcons** [Drive4ik/simple-tab-groups#363]
**Describe the bug**
When I opened the Firefox I have noticed that Simple tab groups extension does not load the tabs groups (also my previously defined tab groups is not visible). When I tried to restore the tabs via copy of the file "storage.js.migrated" at the "browser-extension-data" folder to file "simple-tab-groups@drive4ik.xpi" at "extensions" folder I got an error:

> Access to the file was denied
> 
> The file at moz-extension://f665143a-0f04-49df-b5e3-3ea9de387fc2/popup/popup.html is not readable.
>     It may have been removed, moved, or file permissions may be preventing access.

So I have reinstalled the extension and created the new tab and looked at the file simple-tab-groups@drive4ik.xpi and compared it with backup file storage.js.migrated. I have noticed that The format of the file has been changed and simple-tab-groups@drive4ik.xpi contains the symbols like   which does not contain the backup file. See the [screenshot](https://i.imgur.com/A5wsISK.png). As there is no option to convert between files I can't use my saved tabs.

**To Reproduce**
1. You need previously saved tabs (around February 29, 2019)
2. Try to open them today (April 01, 2019) or later.

**Expected behavior**
There is no tabs defined.

**Desktop (please complete the following information):**
 - OS and version: [e.g. Windows 7 64bit]
 - Firefox version: [e.g. 66.02 64bit]
 - Simple Tab Groups version: [e.g. 3.4.2.1]

**Additional context**
No additional information.

**Comment by techcons** [comment]
I have restored the tabs from backup. This is not the best option, but it have worked for me. Please red the issue and if it's not important information for you please close it.
Thanks.

**Comment by Drive4ik** [comment]
Firefox will no longer store data in `storage.js` files. Many other problems also occurred due to this migration ...
https://www.ghacks.net/2019/02/16/firefox-extensions-to-use-different-storage-type-in-firefox-66/

**Comment by slovdahl** [comment]
I'm probably experiencing the same problem, and I'm not really sure how to solve it. After starting Firefox after updating to 66, I can't do anything else than navigate on the page that happened to be the last one I was on before shutting Firefox 65 down. Reverting to the profile from 65 and downgrading the package works fine, but do you have any suggestions for getting a working 66 up and running?

**Comment by Drive4ik** [comment]
@slovdahl Create backup from addon settings-> delete addon -> restart browser -> install addon -> go to addon settings -> restore latest backup

**Comment by slovdahl** [comment]
Curious.. I did that, but still the same problem. So I guess I'm not having the same problem after all. Sigh.

**Comment by Drive4ik** [comment]
Most likely you need `close firefox 65` -> `open firefox 66` -> install addon -> go to addon settings -> restore latest backup

**Comment by slovdahl** [comment]
Sorry for the false alarm, and thanks for helping out anyway @Drive4ik. I found the problem elsewhere.

Just as an FYI in the unlikely case anyone else experiences the same problem as I did: the cause of my problems was https://forum.manjaro.org/t/howto-enable-tab-switching-in-firefox-using-mouse-wheel/39954. So after removing those two files, updating Firefox to 66 worked fine without removing simple-tab-groups or any other extension. FWIW, it's also possible to fix keep the tab switch mouse using mouse wheel hack, reddit to the rescue: https://www.reddit.com/r/firefox/comments/afh6th/howto_enable_tab_switching_in_firefox_using_mouse/eixtg18/.
